app/license.go

```diff
@@ -12,6 +13,8 @@ import (
 	"github.com/mattermost/mattermost-server/v5/utils"
 )
 
+const requestTrialURL = "https://portal.test.cloud.mattermost.com/api/v1/trials"
```

Comment:
not sure if there is a convention or not in MM for consts to be upper case (or maybe only if they are used in more than one place?)

Reply:
Guess is for testing?  do we have a live server?

Reply:
The testing server is up and running and the plan is to change this urls after QA tests.

The constans case styling is going to be changed for 6.0 for sure. I'm ok moving it back to uppercase.